applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -327,6 +330,9 @@ static bool grant_send(enum coneval_supported_data_type type,
 			goto grant;
 		}
 		break;
+	case WIFI_ACCESS_POINTS:
+		/* TODO: Do we need something here */
+		break;
```

Comment:
Should we just remove this or add own energy threshold parameters for Wi-Fi?
@simensrostad

Reply:
There is currently an ongoing discussion on whether we should continue to support coneval. So I wouldn't care to implement coneval filtering for wifi for now

Reply:
Yep, I'll remove this then.